MonstieWash/Assets/Scripts/Controls/InputManager.cs

```diff
@@ -33,10 +34,20 @@ public class InputManager : MonoBehaviour
     public event Action OnScan_Started;
     public event Action OnScan;
     public event Action OnScan_Ended;
-    #endregion
+	#endregion
+
+	#region MenuActions
+	public event Action OnCancel_Started;
+	public event Action OnCancel;
+	public event Action OnCancel_Ended;
+
+	public event Action OnRightClick_Started;
+	public event Action OnRightClick;
+	public event Action OnRightClick_Ended;
```

Comment:
The only thing i'd change here is naming this action something more general and action-focused than rightclick since we are playing on controller as well as mkb. somthing like OnSkip or even OnAltSelect like it was before should work - whatever you think fits best for what this is used for.